modules/ml/src/main/java/org/apache/ignite/ml/math/functions/IgniteFunction.java

```diff
@@ -26,5 +27,22 @@
  * @see java.util.function.Function
  */
 public interface IgniteFunction<T, R> extends Function<T, R>, Serializable {
-
+    static <T, R> IgniteFunction<T, R> constant(R r) {
```

Comment:
Do we actually need it?

Reply:
Yes, I guess, because it is standard functional block which raises readability.